target/linux/ath79/dts/qca9531_comfast_cf-e314n-v2.dts

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-e314n-v2", "qca,qca9531";
+	model = "COMFAST CF-E314N v2";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wan;
+		led-failsafe = &wan;
+		led-upgrade = &wan;
```

Comment:
If there's no LED to display SYS-activity, we do not have to come up with one. So rather drop this than to misuse the WAN led.

Reply:
For the TP-Link CPE devices we consistently used RSSI LED 4 (max) for that:
https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9533_tplink_cpe210.dtsi#L44

In contrast to LAN/WAN, a single RSSI led blinking cannot be mistaken so easily.

Reply:
I'm aware of that (and i don't like this more).

Assigning a LED to multiple functions makes them ambiguos.
For WLAN / LAN LEDs, i see this is a big nono, as e.g. failsafe can't be displayed clearly.

However, if we already have this ambiguity there, you can also introduce it here.

Reply:
actually, there is a "system" led on the board. Not sure how to figure out the GPIO number thought

Reply:
actually, there is a "system" led on the board. Not sure how to figure out the GPIO number thought

Reply:
It might also happen that you cannot access the LED at all. That's the case for the CPE devices mentioned earlier, they have a system/power which only can be on...

Reply:
so we keep it as is ?

Reply:
so we keep it as is ?

Reply:
If I understood @blocktrron correctly, you can either use rssi led max as we did for the TP-Link CPE devices or you can use no system led at all (and remove the aliases). But you shouldn't keep WAN.